### ISSUE TITLE
[Bug]: Should have accessibility and manual core

### ISSUE THREAD

**Issue by stampyzfanz** [stampyzfanz/TestingGithubActions#35]
### Most appropriate sub-area of TestingGithubActions?

- [X] Accessibility
- [ ] Build
- [ ] Core
